pkg/operator/topology_transition_controller/test_helpers_test.go

```
@@ -0,0 +1,340 @@
+	}
+}
+
+func newTestController(infra *configv1.Infrastructure, conditions []operatorv1.OperatorCondition, transitions []TransitionDescriptor, reconciliationChecks []func(context.Context) (bool, error)) *TopologyTransitionController {
+	return newTestControllerWithClock(infra, conditions, transitions, reconciliationChecks, clocktesting.NewFakePassiveClock(time.Now()))
+}
+
+func newTestControllerWithClock(infra *configv1.Infrastructure, conditions []operatorv1.OperatorCondition, transitions []TransitionDescriptor, reconciliationChecks []func(context.Context) (bool, error), clk *clocktesting.FakePassiveClock) *TopologyTransitionController {
+	fakeConfigClient := configfakeclient.NewSimpleClientset(infra)
+
+	indexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{})
+	indexer.Add(infra)
```

golangci-lint (2.12.2)

[error] Error return value of `indexer.Add` is not checked

(errcheck)